### PR TITLE
v1.6.2 geoips_clavrx - VERSION RELEASE - initial release

### PR DESCRIPTION
**Pull request by mindyls**
# Related Issues
required to close NRLMMD-GEOIPS/geoips#99

# Reviewer Instructions
* Please confirm most recent completed pull request is labeled "DEVELOPMENTAL UPDATE" or "VERSION RELEASE":
    * https://github.com/NRLMMD-GEOIPS/geoips_clavrx/pulls?q=is%3Apr+is%3Aclosed
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates.

# Summary
Initial release - basic reader and product output.

See NRLMMD-GEOIPS/geoips#99 for additional repositories included in this update.

# Testing Instructions
Install GEOIPS and clone v1.6.2-release branch for:
* geoips_clavrx
* test_data_clavrx

Tests should return 0:
$GEOIPS_PACKAGES_DIR/geoips_clavrx/tests/test_all.sh

# Output

# Post Merge Steps
Once this PR has been merged, v1.6.2 will be tagged/released on the geoips_clavrx main branch.

### CODE REVIEW

**Comment by lance-wilson** [comment]
All image-producing tests ran successfully for me (see example below).  The flake8 part of check-code produced a lot of messages about function and variable name capitalization, but all other test scripts returned zero.  Output diff images look okay.
20201201 090000 him8 ahi cldHeight himawari8 45p74 cira 10p0

**Comment by mindyls** [comment]
Interestingly I did not see errors from flake8 regarding capitalization issues (there was an issue with the shebang in the algorithm scripts).

I am going to say we can go ahead and merge this, and we may need to address the flake8 configuration separately (perhaps there are conflicting config files / command line options in your environment ?)

File: setup.py [hunk 1]
```diff
@@ -12,66 +12,89 @@
...
+) as version_file:
     version = version_file.read().strip()
 
-# Info on Python package_data: https://setuptools.pypa.io/en/latest/userguide/datafiles.html
-# Info on Python entry points: https://setuptools.pypa.io/en/latest/userguide/entry_point.html
+# Info on Python package_data:
+# https://setuptools.pypa.io/en/latest/userguide/datafiles.html
+# Info on Python entry points:
+# https://setuptools.pypa.io/en/latest/userguide/entry_point.html
 setuptools.setup(
```
Comment (jsolbrig) [inline setup.py:33]: Should this contain `install_requires=['geoips>=1.6.2']`?

**Review by jsolbrig — CHANGES_REQUESTED** [review]
I'm wondering whether this should required `geoips>=1.6.2` in setup.py. Other than that, everything looks good.

File: setup.py (hunk 1 above)
Comment (jsolbrig) [reply setup.py:33]: After discussion with Mindy, I think we can skip this for now.

**Review by jsolbrig — APPROVED** [review]
After discussion with Mindy, I think we can leave the dependence on `geoips>=1.6.2` out of setup.py.

This looks good to me and all tests passed except for a flake8 error. Approved.

**Pull request merged**